src/pages/openrefine_history.md

```diff
@@ -0,0 +1,60 @@
+---
+title: OpenRefine Project History
+description: A brief overview of OpenRefine History
+hide_table_of_contents: false
+---
+# OpenRefine History 
+
+Over the last ten years, OpenRefine has undergone significant transformations, including changes in its name and development model. It has transitioned from an enterprise-led model to a grant-based funding model.
```

Comment:
I agree with Tom's suggestion, but I think it's still worth mentioning grants because they do have a big impact on what gets worked on (I would say). Perhaps something like this? "It has transitioned from an enterprise-led open source model to a community-based open source model, with later support from occasional grants."

Reply:
I updated the text accordingly.